collectors/VCenterPropertiesCollector.py

```diff
@@ -0,0 +1,53 @@
+from BaseCollector import BaseCollector
+import os, time
+from prometheus_client.core import GaugeMetricFamily
+from tools.Resources import Resources
+from tools.YamlRead import YamlRead
+
+
+class VcenterPropertiesCollector(BaseCollector):
+    def __init__(self):
+        self.iteration = 0
+        while not self.iteration:
+            time.sleep(5)
+            self.get_iteration()
+            print("waiting for initial iteration")
+        print("done: initial query")
```

Comment:
Use the new `wait_for_inventory_data()` method here after your update with the master branch.